jenkins/ci.suse.de/cloud-gating.yaml

```diff
@@ -87,6 +87,7 @@
           cloudsource: stagingcloud8
           ses_enabled: false
           update_after_deploy: false
+          tempest_filter_list: "smoke"
```

Comment:
We don't have yet the ability to run tempest filters in Crowbar 9, that effort is still WIP. Until that is done, we just have to rely on the default behavior (which already is running smoke tempest).